src/java.xml/share/conf/jaxp.properties

```diff
@@ -0,0 +1,180 @@
+################################################################################
+#           XML Library (java.xml) Configuration File
```

Comment:
The file is jaxp.properties and the module is Java APIs for XML Processing. The use of "XML Library (java.xml)" is confusing here, I'm concerned that it will be inconsistent with all the existing docs.

Reply:
"JAXP Configuration File" is now used throughout the spec.